frontend/src/utils/render_utils.jsx

```diff
@@ -0,0 +1,51 @@
+import { useLoader } from '@react-three/fiber';
+// import React from 'react';
+import * as THREE from 'three';
+import { GLTFLoader } from 'three/examples/jsm/loaders/GLTFLoader';
+
+const tileConfig = {
+    width: 1,
+    height: 0.25,
+};
+
+// Converts map coordinates to render coordinates.
+function convertToRenderCoordinates(mapVector, offset = {x: 0, y: 0, z: 0}) {
+    return new THREE.Vector3(
+        mapVector.x * tileConfig.width - offset.x,
+        mapVector.z * tileConfig.height,
+        mapVector.y * tileConfig.width - offset.y
+    )
+};
+
+// Produces the geometry for a map tile.
+export function renderTile(tile) {
```

Comment:
I don't think this one needs to be in here. It should only be used by the map viewer right?

Reply:
I thought it was pretty similar to `objectToComponent`, which is also open to change